### ISSUE TITLE
IDEA: WebUI for autorestic

### ISSUE THREAD

**Issue by r3nor** [cupcakearmy/autorestic#254]
It would be nice to have a (very simple) webui for autorestic. Maybe this features should be available:

- Configure a new backup / edit existing ones.
- Explore a snapshot / repo
- Recover files

**Comment by cupcakearmy** [comment]
Love the idea, however as you can see there are a lot of issues that rn i don't have time to fix :/ and they need to be addressed first.

A good starting point maybe would be an editor for the config file, that would be kind of neat :)

**Comment by patrick-motard** [comment]
@cupcakearmy would you be open to contributions?

**Comment by 12nick12** [comment]
There's [THIS](https://github.com/emuell/restic-browser), as long as you're using one of the supported backends it works great if you just need to browse. A WebUI would be cool, but this works for now and once you find what you need on your PC you can just tell autorestic what exactly you need to restore via CLI.

**Comment by cupcakearmy** [comment]
Yes, I'm def. open to contributors! However I kind of want to do a major cleanup.
The first step would be have a jsonschema for the config file. this would allow validation and autocompletion in editors, From that we could easily spin up a small web ui to edit/generate/do whatever thing. Also for sharing maybe? the possibilities are endless :)
